model/article-model.js

```diff
@@ -64,3 +64,14 @@ exports.insertComments = (username, body, article_id) => {
   })
 }
 
+  exports.updateArticle = (votes, id) => {
+  return db.query(`
+  UPDATE articles
+  SET votes = votes + $1
+  WHERE article_id = $2
+  RETURNING *;
+  `, [votes, id])
+  .then(({rows})=> {
+    return rows[0]
```

Comment:
Do you think an custom error would be needed in here if the article ID isn't in the database?